compact/far_forward/electron_beamline.xml

```diff
@@ -0,0 +1,116 @@
+
+    <!-- Q0eF quadrupole magnet -->
+    <constant name="Q0eF_StartZ" value="7000.*mm"/>
+    <constant name="Q0eF_EndZ" value="5800.*mm"/>
+    <constant name="Q0eF_InnerRadius" value="25.*mm"/>
+
+    <!-- Q1eF quadrupole magnet -->
+    <constant name="Q1eF_StartZ" value="12370.*mm"/>
+    <constant name="Q1eF_EndZ" value="10760.*mm"/>
+    <constant name="Q1eF_InnerRadius" value="63*mm"/>
+
+    <!-- beam section at forward region to +15 meters -->
+    <constant name="cen_pos_fwd_zmax" value="15000.*mm"/>
+
+  </define>
+
+  <detectors>
+
+    <!-- beam from central section at positive z to Q0eF -->
+    <detector name="cen_beam_Q0" type="ConeBeamEl"
+      rmin0="cen_pos_r1-cen_pos_wall_dr"
+      rmax0="cen_pos_r1"
+      rmin1="Q0eF_InnerRadius"
+      rmax1="Q0eF_InnerRadius+cen_pos_wall_dr"
+      dz="Q0eF_EndZ-cen_pos_zmax"
+      zpos="(Q0eF_EndZ+cen_pos_zmax)/2.">
+    </detector>
+
+    <!-- Q0eF magnet -->
+    <detector name="Q0eF" type="ip6_CylindricalDipoleMagnet" vis="RedVis">
+      <placement  x="0" y="0" z="(Q0eF_StartZ+Q0eF_EndZ)/2." theta="0"/>
+      <dimensions x="Q0eF_InnerRadius*4" y="Q0eF_InnerRadius*4" z="Q0eF_StartZ-Q0eF_EndZ" r="2.0*Q0eF_InnerRadius" />
+      <apperture  x="Q0eF_InnerRadius*2" y="Q0eF_InnerRadius*2" r="Q0eF_InnerRadius" />
+      <coil dx="2*cm" dy="1.5*cm" />!--unchecked--
+    </detector>
+
+    <!-- inner vacuum for Q0eF -->
+    <detector name="Q0eF_vac" type="DD4hep_TubeSegment" vis="VisFwElInvisible">
+      <material name="Vacuum"/>
+      <tubs rmin="0" rmax="Q0eF_InnerRadius" zhalf="(Q0eF_StartZ-Q0eF_EndZ)/2."/>
+      <position x="0" y="0" z="(Q0eF_StartZ+Q0eF_EndZ)/2."/>
+      <rotation x="0" y="0" z="0"/>
+    </detector>
+
+    <!-- beam between Q0eF and Q1eF magnets -->
+    <detector name="cen_beam_Q01" type="ConeBeamEl"
+      rmin0="Q0eF_InnerRadius"
+      rmax0="Q0eF_InnerRadius+cen_pos_wall_dr"
+      rmin1="Q1eF_InnerRadius"
+      rmax1="Q1eF_InnerRadius+cen_pos_wall_dr"
+      dz="Q1eF_EndZ-Q0eF_StartZ"
+      zpos="(Q1eF_EndZ+Q0eF_StartZ)/2.">
+    </detector>
+
+    <!-- Q1eF magnet -->
+    <detector name="Q1eF" type="ip6_CylindricalDipoleMagnet" vis="RedVis">
+      <placement  x="0" y="0" z="(Q1eF_StartZ+Q1eF_EndZ)/2." theta="0"/>
+      <dimensions x="Q1eF_InnerRadius*4" y="Q1eF_InnerRadius*4" z="Q1eF_StartZ-Q1eF_EndZ" r="2.0*Q1eF_InnerRadius" />
+      <apperture  x="Q1eF_InnerRadius*2" y="Q1eF_InnerRadius*2" r="Q1eF_InnerRadius" />
+      <coil dx="2*cm" dy="1.5*cm" />!--unchecked--
+    </detector>
+
+    <!-- inner vacuum for Q1eF -->
+    <detector name="Q1eF_vac" type="DD4hep_TubeSegment" vis="VisFwElInvisible">
+      <material name="Vacuum"/>
+      <tubs rmin="0" rmax="Q1eF_InnerRadius" zhalf="(Q1eF_StartZ-Q1eF_EndZ)/2."/>
+      <position x="0" y="0" z="(Q1eF_StartZ+Q1eF_EndZ)/2."/>
+      <rotation x="0" y="0" z="0"/>
+    </detector>
+
+    <!-- beam section at forward region to +15 meters -->
+    <detector name="cen_beam_fwd" type="ConeBeamEl"
```

Comment:
Can we avoid creating yet another beam pipe plugin? Can't this be done with https://github.com/eic/epic/blob/main/src/BackwardsBeamPipe_geo.cpp (maybe rename it to `BeamPipe_geo.cpp`)? All these extra source files are more work to keep up to date later.

Reply:
Will be addressed

Reply:
Has been tried, no success in implementing via suggested class.

Reply:
Can you add more detail about why it didn't work?

Reply:
I was not able to recreate the geometry using the proposed class. I will look into it more this week.

Reply:
Thank you Simon for your suggestion! I will look into it.

Reply:
Implemented, geometry should check out. Thank you for the tip!